ParalayoutTests/AngleTests.swift

```diff
@@ -81,14 +113,14 @@ final class AngleTests: XCTestCase {
             // Test two equivalent values for the same angle.
             assertEqual(
                 Angle(degrees: angleDegrees).point(atDistance: distance, from: centerPoint),
-                position.point(in: rect),
+                position.point(in: rect, layoutDirection: .leftToRight),
                 accuracy: 1e-4,
                 file: file,
                 line: line
             )
             assertEqual(
                 Angle(degrees: angleDegrees - 360).point(atDistance: distance, from: centerPoint),
-                position.point(in: rect),
+                position.point(in: rect, layoutDirection: .leftToRight),
```

Comment:
No surprise, dealing with layout directions makes your layout code more verbose, since everything dealing with positions has to be able to resolve the leading/trailing values.

To make this less verbose, we could expose `ResolvedPosition` and add variants of all these methods that take a resolved position directly instead of a raw position plus the layout direction. There's definitely some trade-offs there around consumers accidentally resolving the positions incorrectly. Not totally sure what the right path here is.